pkg/collector/stats/stats.go

```diff
@@ -20,20 +20,35 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/sustainable-computing-io/kepler/pkg/bpf"
 	"github.com/sustainable-computing-io/kepler/pkg/collector/stats/types"
 	"github.com/sustainable-computing-io/kepler/pkg/config"
 	acc "github.com/sustainable-computing-io/kepler/pkg/sensors/accelerator"
 	"k8s.io/klog/v2"
 )
 
 var (
+	// Default metric sets. In addition, each metric set is used by the model
```

Comment:
For the moment, need to leave these as separate sets as they are used by the model to determine the stats groups to update (dynamic/idle/...)